qutebrowser/config/config.py

```diff
@@ -665,16 +674,21 @@ def _get_stylesheet(self) -> str:
         """
         return _render_stylesheet(self._stylesheet)
 
-    @pyqtSlot()
     def _update_stylesheet(self) -> None:
```

Comment:
Since this is only one line and only called from `_update_stylesheet_conditional` I don't see the benefit of having another method - just call `self._obj.setStyleSheet` in `_update_stylesheet_conditional` directly.

Reply:
May be a good idea. This is private interface and it can be refactor out again if convenient.